addons/pos_stripe/models/pos_payment_method.py

```diff
@@ -83,6 +84,8 @@ def stripe_payment_intent(self, amount):
 
         if resp.ok:
             return resp.json()
+        else:
+            return json.loads(resp.text)
```

Comment:
What's the difference between `resp.text` and `resp.json()`? And I think this change should be in the `stripe_capture_payment` because that's where you put the added check in the js code.